evm_arithmetization/src/cpu/kernel/tests/add11.rs

```diff
@@ -1,326 +1,341 @@
+        block_number: 1.into(),
+        block_difficulty: 0x020000.into(),
+        block_random: H256::from_uint(&0x020000.into()),
+        block_gaslimit: 0xff112233u32.into(),
+        block_chain_id: 1.into(),
+        block_base_fee: 0xa.into(),
+        block_gas_used: 0xa868u64.into(),
+        block_bloom: [0.into(); 8],
+    };
+
+    let expected_state_smt_after = {
+        let mut smt = Smt::<MemoryDb>::default();
+        let beneficiary_account_after = AccountRlp {
+            nonce: 1.into(),
+            ..AccountRlp::default()
+        };
+        let sender_account_after = AccountRlp {
+            balance: 0xde0b6b3a75be550u64.into(),
+            nonce: 1.into(),
+            ..AccountRlp::default()
+        };
+        let to_account_after = AccountRlp {
+            balance: 0xde0b6b3a76586a0u64.into(),
+            code_hash,
+            ..AccountRlp::default()
+        };
+
+        set_account(
+            &mut smt,
+            H160(beneficiary),
+            &beneficiary_account_after,
+            &HashMap::new(),
+        );
+        set_account(
+            &mut smt,
+            H160(sender),
+            &sender_account_after,
+            &HashMap::new(),
+        );
+        set_account(
+            &mut smt,
+            H160(to),
+            &to_account_after,
+            &HashMap::from([(U256::zero(), 2.into())]), // Storage map: { 0 => 2 }
+        );
+
+        smt
+    };
+
+    let receipt_0 = LegacyReceiptRlp {
+        status: true,
+        cum_gas_used: 0xa868u64.into(),
+        bloom: vec![0; 256].into(),
+        logs: vec![],
+    };
+    let mut receipts_trie = HashedPartialTrie::from(Node::Empty);
+    receipts_trie
+        .insert(
+            Nibbles::from_str("0x80").unwrap(),
+            rlp::encode(&receipt_0).to_vec(),
+        )
+        .unwrap();
+    let transactions_trie: HashedPartialTrie = Node::Leaf {
+        nibbles: Nibbles::from_str("0x80").unwrap(),
+        value: txn.to_vec(),
+    }
+    .into();
+
+    let trie_roots_after = TrieRoots {
+        state_root: H256::from_uint(&hashout2u(expected_state_smt_after.root)),
+        transactions_root: transactions_trie.hash(),
+        receipts_root: receipts_trie.hash(),
+    };
+    let inputs = GenerationInputs {
+        signed_txn: Some(txn.to_vec()),
+        withdrawals: vec![],
+        tries: tries_before,
+        trie_roots_after,
+        contract_code,
+        block_metadata,
+        checkpoint_state_trie_root: HashedPartialTrie::from(Node::Empty).hash(),
+        txn_number_before: 0.into(),
+        gas_used_before: 0.into(),
+        gas_used_after: 0xa868u64.into(),
+        block_hashes: BlockHashes {
+            prev_hashes: vec![H256::default(); 256],
+            cur_hash: H256::default(),
+        },
+    };
+
+    let initial_stack = vec![];
+    let mut interpreter: Interpreter<F> =
+        Interpreter::new_with_generation_inputs(0, initial_stack, inputs);
```

Comment:
0 should be `KERNEL.global_labels["main"]`?

Reply:
Yeah I've replaced it for clarity.